schema_test_values.yaml

```diff
@@ -1,13 +1,2 @@
 properties:
   # set a default value (need to copy other fields for validation to pass)
-  enterpriseLicense:
```

Comment:
We can just delete this file and delete this line in the dockerfile
```Dockerfile
COPY schema_test_values.yaml /data-test/schema.yaml
```

Reply:
Yeah good call, updated and confirmed the verification still passes